test/functional/rpc_getblockstats.py

```diff
@@ -49,6 +50,13 @@ def generate_test_data(self, filename):
         self.sync_all()
 
         self.nodes[0].sendtoaddress(address=address, amount=10, subtractfeefromamount=True)
+
+        inputs = self.nodes[0].listunspent()[:3]
+        outputs = {self.nodes[0].getnewaddress():1,self.nodes[0].getnewaddress():2,self.nodes[0].getnewaddress():sum(i["amount"] for i in inputs)-Decimal("3.001")}
+        rawtx = self.nodes[0].createrawtransaction(inputs, outputs, 0, True)
```

Comment:
Can you use named arguments here (`createrawtransaction(inputs=inputs, outputs=outputs, locktime=0, replaceable=True)`) so it's clear that this tx is signaling opt-in RBF. Maybe also add a comment above saying that this tx is signaling opt-in RBF.